chaoslib/pumba/pod_failure_by_sigkill.yml

```diff
@@ -23,7 +51,7 @@
       # the grep filters the output to just "Ready" nodes, and the awk returns just the node names in the single-line format: "node01" "node02" "node03" "node04"
       # This list is then passed into the Go-template output formatter, so that the main 'kubectl get pod...' 
       # command only returns the status of the pods on the Ready nodes (e.g. {{if eq .spec.nodeName "node01" "node02" "node03" "node04"}}{{.status.phase}})
-    - name: Confirm that the pumba ds is running on all nodes
+    - name: Confirm that the pumba job is running on {{ app_node }} nodes
```

Comment:
Previously, We are putting  `ignore_errors: true` for this task because we are unable to check the running status of the DS. But now it is changed to Job so it is easy to check the status so can you please update this task and remove ignore_error for this task?

Reply:
Done